azure-pipelines.yml

```diff
@@ -71,57 +71,57 @@ jobs:
     matrix:
       Python37-64bit + OPTIONAL_DEPS:
         python.version: '3.7'
-        EXTRA_DEPENDS: "scikit_learn pandas statsmodels==0.9.0 tables scipy==1.1"
+        EXTRA_DEPENDS: "scikit_learn pandas statsmodels tables scipy"
       Python37-64bit:
         python.version: '3.7'
-      Python38-64bit:
-        python.version: '3.8'
-      CONDA Python37-64bit + OPTIONAL_DEPS:
-        python.version: '3.7'
-        EXTRA_DEPENDS: "scikit-learn pandas statsmodels==0.9.0 pytables scipy==1.1"
-        INSTALL_TYPE: "conda"
-      CONDA Python37-64bit:
-        python.version: '3.7'
-        INSTALL_TYPE: "conda"
-      CONDA Python36-64bit:
-        python.version: '3.6'
-        INSTALL_TYPE: "conda"
-      Python37-64bit + VIZ:
-        TEST_WITH_XVFB: "1"
-        python.version: '3.7'
-        MESA_GL_VERSION_OVERRIDE: '3.3'
-        LIBGL_ALWAYS_INDIRECT: 'y'
-        EXTRA_DEPENDS: "scikit_learn vtk fury scipy xvfbwrapper"
-
-- template: ci/azure/windows.yml
-  parameters:
-    name: Windows
-    vmImage: vs2017-win2016
-    matrix:
-      Python37-64bit + OPTIONAL_DEPS:
-        python.version: '3.7'
-        EXTRA_DEPENDS: "scikit_learn pandas statsmodels==0.9.0 tables scipy==1.1"
       # Python38-64bit:
       #   python.version: '3.8'
-      Python37-64bit:
-        python.version: '3.7'
-      Python36-64bit:
-        python.version: '3.6'
-      # Python35-64bit:
-      #   python.version: '3.5'
       CONDA Python37-64bit + OPTIONAL_DEPS:
         python.version: '3.7'
-        EXTRA_DEPENDS: "scikit-learn pandas statsmodels==0.9.0 pytables scipy==1.1"
+        EXTRA_DEPENDS: "scikit-learn pandas statsmodels pytables scipy==1.2"
         INSTALL_TYPE: "conda"
-      CONDA Python37-64bit:
-        python.version: '3.7'
-        INSTALL_TYPE: "conda"
-      CONDA Python36-64bit:
-        python.version: '3.6'
-        INSTALL_TYPE: "conda"
-      Python37-64bit + VIZ:
-        TEST_WITH_XVFB: "1"
-        python.version: '3.7'
-        MESA_GL_VERSION_OVERRIDE: '3.3'
-        LIBGL_ALWAYS_INDIRECT: 'y'
-        EXTRA_DEPENDS: "scikit_learn vtk fury scipy"
+      # CONDA Python37-64bit:
+      #   python.version: '3.7'
+      #   INSTALL_TYPE: "conda"
+      # CONDA Python36-64bit:
+      #   python.version: '3.6'
+      #   INSTALL_TYPE: "conda"
+      # Python37-64bit + VIZ:
+      #   TEST_WITH_XVFB: "1"
+      #   python.version: '3.7'
+      #   MESA_GL_VERSION_OVERRIDE: '3.3'
+      #   LIBGL_ALWAYS_INDIRECT: 'y'
+      #   EXTRA_DEPENDS: "scikit_learn vtk fury scipy xvfbwrapper"
+
+# - template: ci/azure/windows.yml
```

Comment:
Many comments added here. Is that what you want to do. Disable some bots? Please enlighten us.

Reply:
I use this PR to understand @BramshQamar issue with CI's. All these comments are just to accelerate the build process so they are temporary. I will remove them soon.